satpy/readers/ami_l1b.py

```diff
@@ -182,32 +163,55 @@ def get_dataset(self, dataset_id, ds_info):
         # only take "no error" pixels as valid
         data = data.where(qf == 0)
 
-        channel_name = attrs.get('channel_name', dataset_id.name)
-        coeffs = CALIBRATION_COEFFS.get(channel_name)
-        if coeffs is None and dataset_id.calibration is not None:
-            raise ValueError("No coefficients configured for {}".format(dataset_id))
+        # Calibration values from file, fall back to built-in if unavailable
+        gain = self.nc.attrs['DN_to_Radiance_Gain']
+        offset = self.nc.attrs['DN_to_Radiance_Offset']
+
         if dataset_id.calibration in ('radiance', 'reflectance', 'brightness_temperature'):
-            gain = coeffs[0]
-            offset = coeffs[1]
             data = gain * data + offset
         if dataset_id.calibration == 'reflectance':
             # depends on the radiance calibration above
-            rad_to_alb = coeffs[2]
+            rad_to_alb = self.nc.attrs['Radiance_to_Albedo_c']
             if ds_info.get('units') == '%':
                 rad_to_alb *= 100
             data = data * rad_to_alb
         elif dataset_id.calibration == 'brightness_temperature':
-            # depends on the radiance calibration above
-            # Convert um to m^-1 (SI units for pyspectral)
-            wn = 1 / (dataset_id.wavelength[1] / 1e6)
-            # Convert cm^-1 (wavenumbers) and (mW/m^2)/(str/cm^-1) (radiance data)
-            # to SI units m^-1, mW*m^-3*str^-1.
-            bt_data = rad2temp(wn, data.data * 1e-5)
-            if isinstance(bt_data, np.ndarray):
-                # old versions of pyspectral produce numpy arrays
-                data.data = da.from_array(bt_data, chunks=data.data.chunks)
+            print(self.calib_mode)
```

Comment:
Left over print

Reply:
Removed

Reply:
The github page is showing that this print still exists.